go/vt/vtgate/planbuilder/operators/rewriters.go

```diff
@@ -218,6 +218,9 @@ func bottomUp(
 			childID = childID.Merge(resolveID(oldInputs[0]))
 		}
 		in, changed := bottomUp(operator, childID, resolveID, rewriter, shouldVisit, false)
+		if DebugOperatorTree && changed.Changed() {
+			fmt.Println(ToTree(in))
```

Comment:
Looks like some stray debugging here that's left?

Reply:
To be clear, this will only be logged from testing code, not production code.